### DISCUSSION TITLE
Vcxproj generator

### DISCUSSION THREAD

**Discussion by olizit** [mesonbuild/meson#11568] [Q&A]
I use meson build for a c++ project with the default ninja backend. From this, I want to generate a Visual Studio makefile project vcxproj file to edit, build (with meson thanks to make file project type) and debug in Visual Studio. 
Is it a way to generate vcxproj  file ?
Or at least how can I get all dependencies files (cpp files and include files) to create a vcxproj file ?

**Comment by eli-schwartz** [comment]
Meson supports a VS backend, so you can do that directly. It will do a native one, not one wrapping ninja.

If you want a VS solution that wraps ninja, someone is working on one in #11049
